### DISCUSSION TITLE
Changing Modal's close button styling

### DISCUSSION THREAD

**Discussion by mbeldartwilio** [twilio-labs/paste#2297] [General] [answered]
Working Modal but not able to change the close button background to transparent or white :confused:
Can anyone help?
![image](https://user-images.githubusercontent.com/90804139/157970444-cfe88b71-0fa8-49af-ab82-bad745ead9d2.png)

**Comment by nkrantz** [comment, marked as answer]
Hi @mbeldartwilio - could you please share the code you're using or a link to a codesandbox with the problem so that I can take a look? Thanks!

**Reply by nkrantz** [reply]
Hi again @mbeldartwilio - just checking in to see whether you're still having trouble with the Modal close button. Let us know!

**Reply by shleewhite** [reply]
Following up again @mbeldartwilio , do you still need help with this?

**Reply by mbeldartwilio** [reply]
@shleewhite @nkrantz apologies for late response. 
We decided to go with different approach. Thanks.
